blog/_posts/2022-07-13-scala-improvement-process-reloaded.md

```diff
@@ -21,17 +21,14 @@ It is our pleasure to present the new SIP Committee members:
 
 * Björn Regnell, Lund University
 * Chris Andrews, Morgan Stanley
-* Gabriele Petronella, buildo
 * Guillaume Martres, EPFL
-* Iulian Dragos, Databricks
+* Haoyi Li, Databricks
```

Comment:
Li Haoyi is not in Databricks anymore or am I mistaken ?

Reply:
Do you know his current affiliation?